index.html

```diff
@@ -27,15 +27,15 @@
     <meta property="og:type" content="website">
     <meta property="og:locale" content="pt_BR">
     <meta property="og:url" content="https://fatecsorocaba.github.io/semana-da-tecnologia/">
-    <meta property="og:title" content="XXIV Semana da Tecnologia 2017 da Fatec Sorocaba">
+    <meta property="og:title" content="24ª Semana da Tecnologia 2017 da Fatec Sorocaba">
     <meta property="og:description" content="Participe da primeira edição feita por alunos! De 03 a 08 de outubro na Fatec Sorocaba.">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.css" />
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
     <link rel="stylesheet" href="css/style.min.css">
   </head>
   <body>
       <header>
-          <h1>XXIV Semana da Tecnologia - 2017</h1>
+          <h1>24ª Semana da Tecnologia - 2017</h1>
```

Comment:
I think we can keep with XXIV for now, we don't really know yet if is a necessary change.